components/admin_console/group_settings/groups_list/groups_list.jsx

```diff
@@ -32,7 +45,21 @@ export default class GroupsList extends React.PureComponent {
             checked: {},
             loading: true,
             page: 0,
+            showFilters: false,
+            searchString: '',
         };
+        Object.entries(FILTER_STATE_SEARCH_KEY_MAPPING).forEach(([key]) => {
+            this.state[key] = false;
+        });
+    }
+
+    closeFilters = (e) => {
+        const filtersNode = document.getElementById('group-filters');
+        if (!filtersNode || filtersNode.contains(e.target) || (e.target.parentNode && e.target.parentNode.tagName === 'g')) {
```

Comment:
@jespino I don't love looking for the `<g>`. Basically this is a hack because the SVG checkbox isn't available in the DOM when one checks if `filtersNode.contains(e.target)`. Do you happen to know a better way?

Reply:
I think the simplest option here would be handle the click event in the root of `renderSearchFilters`, and simply stop the propagation there. This will stop any click event to reach the document level inside there, which is the final goal.